Services/Database/interfaces/interface.ilTableLockInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Class ilTableLockInterface
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilTableLockInterface {
+
+	/**
+	 * @param bool $lock_bool
+	 * @return ilTableLockInterface
+	 */
+	public function lockSequence($lock_bool);
```

Comment:
Do we really need that parameter? If i understand correctly, the lock per default does not comprise the sequence table and we could switch that behaviour on by using this method. Would we ever want to switch that behaviour off again afterwards?